plugins/modules/idrac_session.py

```diff
@@ -65,10 +65,10 @@
     choices: [present, absent]
     type: str
     default: present
-  auth_token:
```

Comment:
Lets add `auth_token` as alias just to keep the back war compatibility.

Reply:
Done